gallery/workbook/microsoft.insights-components.json

```diff
@@ -258,6 +258,19 @@
 					"kind": "dashboard"
 				}
 			]
+		},
+		{
+			"id": "SDK Stats Dashboards",
+			"name": "SDK Stats Dashboards (Preview)",
+			"templates": [
+				{
+					"id": "Workbooks/SDKStats",
+					"author": "Microsoft",
+					"description": "SDK Stats Insights Dashboard",
+					"name": "SDK Stats Insights",
+					"kind": "dashboard"
```

Comment:
its marked as kind "dashboard" here which will open it in the workbooks preview view. but the content in the workbook doesn't look to be using dashboard mode or layout, so is there a specific reason it is here?

Reply:
I assumed it was more generic terminology. I'll remove given it's not relevant to the display here.